packages/experimental/TabList/src/TabList.tsx

```diff
@@ -39,8 +39,46 @@ export const TabList = compose<TabListType>({
     const Slots = useSlots(userProps, (layer) => tabs.state[layer] || userProps[layer]);
 
     // Return the handler to finish render.
-    return () => {
-      return <Slots.root></Slots.root>;
+    return (final: TabListProps, ...children: React.ReactNode[]) => {
+      if (!tabs.state) {
+        return null;
+      }
+
+      const { defaultTabbableElement, isCircularNavigation, ...mergedProps } = mergeProps(tabs.props, final);
+
+      // Populate the tabsItemKeys array.
+      if (children) {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore - TODO, fix typing error
```

Comment:
any idea what the typing error is here? maybe we can fix it?

Reply:
I think this had to do with the old child param being a ReactChild type. Switching to ReactNode seems to fix the issue.